dateparser/languages/locale.py

```diff
@@ -185,31 +184,10 @@ def translate_search(self, search_string, settings=None):
             original_tokens, simplified_tokens = self._simplify_split_align(sentence, settings=settings)
             translated_chunk = []
             original_chunk = []
-            simplified_tokens_length = len(simplified_tokens)
-            skip_next_token = False
             for i, word in enumerate(simplified_tokens):
-
-                next_word = simplified_tokens[i + 1] if (simplified_tokens_length - 1) > i else ""
-                current_and_next_joined = self._join_chunk([word, next_word], settings=settings)
-
-                if skip_next_token:
-                    skip_next_token = False
-                    continue
-
                 if word == '' or word == ' ':
```

Comment:
Hi @gavishpoddar 
You removed this because it is in another PR, right?

Reply:
Hi @noviluni 
Yes, I have removed this as its now part of PR #953